tests/tracking/test_tracking.py

```diff
@@ -867,56 +867,58 @@ def test_search_runs():
         mlflow.log_param("p2", "aa")
         mlflow.set_tag("t2", "second-tag-val")
 
-    def verify_runs(runs, expected_set):
-        assert set([r.info.run_id for r in runs]) == set([logged_runs[r] for r in expected_set])
-
     experiment_id = MlflowClient().get_experiment_by_name("exp-for-search").experiment_id
 
     # 2 runs in this experiment
     assert len(MlflowClient().list_run_infos(experiment_id, ViewType.ACTIVE_ONLY)) == 2
 
+    def run_and_verify(experiment_ids, query, check, **kwargs):
+        runs = MlflowClient().search_runs(experiment_ids, query, **kwargs)
```

Comment:
Out of curiosity, how are these tests passing for the skinny client? Is it because installing alembic / sqlalchemy / sqlparse is also pulling in pandas? https://github.com/mlflow/mlflow/blob/032ec8841dad13492a6eb3c9acdf78803ed90f76/dev/run-python-skinny-tests.sh#L17